examples/web-react/src/demo.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { Button } from '@lmc-eu/spirit-web-react/src/components/Button';
```

Comment:
I think we should do this better. Like `@lmc-eu/spirit-web-react/components/button` or `@lmc-eu/spirit-web-react/button` eg. get rid of `src` while building package. For better usabillity.

Reply:
Fixed in #18.